data/speakers.ts

```diff
@@ -1025,19 +1025,19 @@ const speakers: Speaker[] = [
   {
     objectID: 'sambego',
     fullName: 'Sam Bellen',
-    tags: ['Web Audio', 'Browser APIs'],
-    email: null,
-    website: null,
+    tags: ['Identity', 'Authentication', 'Web Audio', 'Browser APIs'],
+    email: 'hey@sambego.be',
+    website: 'https://sambego.tech',
     social: {
       twitter: 'sambego',
       github: 'Sambego',
       blog: 'https://blog.sambego.be/',
       linkedin: 'https://www.linkedin.com/in/sambellen/',
     },
     talks: [],
```

Comment:
Hi @Sambego, thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Hi, I'm merging this for now, but could you please update your talks in the new PR? Thank you.